extensions/intellij/intellij_generator_plugin/src/main/java/com/primeholding/rxbloc_generator_plugin/action/BootstrapSingleTestAction.kt

```diff
@@ -466,4 +443,61 @@ void main () {
         }
         return sb.toString()
     }
+
+
+    private fun createGoldenToolkitFileContent(blocFieldCase: String, blocSnakeCase: String): String =
+        """
+import '../../helpers/golden_helper.dart';
+import '../../helpers/models/scenario.dart';
+import '${blocSnakeCase}_factory.dart';
+
+
+void main () {
+  runGoldenTests(
+    [
+      generateDeviceBuilder(
+          widget: ${blocFieldCase}Factory(), //example: Stubs.emptyList
+          scenario: Scenario(name: '${blocSnakeCase}_empty')),
+      generateDeviceBuilder(
+          widget: ${blocFieldCase}Factory(), //example:  Stubs.success
+          scenario: Scenario(name: '${blocSnakeCase}_success')),
+      generateDeviceBuilder(
+          widget: ${blocFieldCase}Factory(), //loading
+          scenario: Scenario(name: '${blocSnakeCase}_loading')),
+      generateDeviceBuilder(
+          widget: ${blocFieldCase}Factory(),
+          scenario: Scenario(name: '${blocSnakeCase}_error'))
+  ]);
+}
+""".trimIndent()
+
+    private fun createAlchemistGoldenFileContent(blocFieldCase: String, blocSnakeCase: String): String =
+        """
+import '../../helpers/golden_helper.dart';
+import '${blocSnakeCase}_factory.dart';
+
+
+void main () {
+  runGoldenTests(
+    [
+      buildScenario(
+        scenario:  '${blocSnakeCase}_empty',
+        widget: ${blocFieldCase}Factory(),
+      ),
+      buildScenario(
+        scenario:  '${blocSnakeCase}_success',
+        widget: ${blocFieldCase}Factory(),
+      ),
+      buildScenario(
+        scenario:  '${blocSnakeCase}_loading',
+        customPumpBeforeTest: animationCustomPump,
```

Comment:
Since we updated our goldens implementation, this line of code is not needed anymore. So we can remove it.